spec/linkedList.js

```diff
@@ -0,0 +1,175 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import LinkedList from '../src/linkedList'
+
+chai.use(chaiChange)
+
+before( () => {
+  let linkedList = new LinkedList()
+  linkedList.insert('apple')
+  linkedList.insert('banana')
+})
+describe('LinkedList', () => {
+
+
+  it('exists', () => {
+
+    expect(LinkedList).to.be.a('function')
+  })
+
+    it('returns head node', () => {
+      const linkedList = new LinkedList()
+      linkedList.insert('apple')
```

Comment:
you used the before hook... that should do this before each test... why are you still doing it for every test?